e2e/test/scenarios/dashboard/dashboard-card-resizing.cy.spec.js

```diff
@@ -0,0 +1,171 @@
+import {
+  addOrUpdateDashboardCard,
+  editDashboard,
+  getDashboardCards,
+  restore,
+  saveDashboard,
+  visitDashboard,
+} from "e2e/support/helpers";
+import { SAMPLE_DB_ID } from "e2e/support/cypress_data";
+import { ORDERS, ORDERS_ID, PEOPLE } from "metabase-types/api/mocks/presets";
+import {
```

Comment:
Not sure if it's okay here; basically we're using the utility in code and in tests; if something is wrong in the implementation these tests won't catch it. I think even duplicating min sizes for each viz type is much better.